ServerCore/Pages/Hints/Edit.cshtml.cs

```diff
@@ -53,6 +53,16 @@ public async Task<IActionResult> OnPostAsync()
             try
             {
                 await _context.SaveChangesAsync();
+
+                var puzzleName = await _context.Hints.Where(m => m.Id == Hint.Id).Select(m => m.Puzzle.Name).FirstOrDefaultAsync();
+
+                var teamMembers = await (from TeamMembers tm in _context.TeamMembers
+                                         join HintStatePerTeam hspt in _context.HintStatePerTeam on tm.Team equals hspt.Team
+                                         where hspt.Hint.Id == Hint.Id
+                                         select tm.Member.Email).ToListAsync();
+                MailHelper.Singleton.SendPlaintextBcc(teamMembers,
+                    $"Hint updated for '{Hint.Description}' on {puzzleName}",
```

Comment:
Should we start all of the subjects with the event name? ex.
"{Event}: Hint updated for..."

Reply:
In the name of brevity, I am only going to do this for registration since we may have parallel registration, but you ought to only be *playing* in one event at a time.

Reply:
I take it back, I just did it everywhere.